crates/rpc/rpc-types/src/eth/engine/payload.rs

```diff
@@ -167,14 +171,13 @@ impl TryFrom<ExecutionPayload> for SealedBlock {
                     .uint_try_to()
                     .map_err(|_| PayloadError::BaseFee(payload.base_fee_per_gas))?,
             ),
+            blob_gas_used: Some(payload.blob_gas_used.unwrap_or_default().as_u64()),
+            excess_blob_gas: Some(payload.excess_blob_gas.unwrap_or_default().as_u64()),
```

Comment:
Similar to the other suggestion, these can be directly mapped now
```suggestion
            blob_gas_used: payload.blob_gas_used.map(|blob_gas_used| blob_gas_used.as_u64()),
            excess_blob_gas: payload
                .excess_blob_gas
                .map(|excess_blob_gas| excess_blob_gas.as_u64()),
```